src/audio/codec_adapter/CMakeLists.txt

```diff
@@ -4,6 +4,15 @@ add_local_sources(sof codec_adapter.c codec/generic.c)
 
 if(CONFIG_CADENCE_CODEC)
 add_local_sources(sof codec/cadence.c)
+
+if(CONFIG_CADENCE_CODEC_MP3_DEC)
+sof_add_static_library(xa_mp3_dec ${CMAKE_CURRENT_LIST_DIR}/imx-audio-codec/core_lib/pilibxa_mp3_dec/lib/xa_mp3_dec.a)
```

Comment:
These directories should be set in Kconfig menu. That way Yocto can ship a .config too.

Reply:
In fact all our static libs should have a $PATH set in Kconfig

Reply:
Is there any standard naming for this? Should be this the path used by all static libs? 

And then we add something like this in CMakeList.txt:

```
 sof_add_static_library(xa_mp3_dec ${STATIC_LIB_PATH}/core_lib/pilibxa_mp3_dec/lib/xa_mp3_dec.a) 
```

Reply:
In fact, I think having the entire binary path set in the Kconfig would make sense. Similar with something like CONFIG_EXTRA_FIRMWARE in the kernel https://cateee.net/lkddb/web-lkddb/EXTRA_FIRMWARE.html

Reply:
Yeah, this would be a $PATH_CODECX so would be per codec. i.e. on the Kconfig menu when we enable the CODEC then the PATH entry box should activate and allow the user to set a PATH. This also allows Yocto and other embedded distros to carry a config with their own PATHs

Reply:
@lgirdwood OK, sent v2